baremetal.go

```diff
@@ -0,0 +1,114 @@
+// Package vagrantssh provides host connectivity in go for system/integration
+// testing in a multi host environment. It supports two testbed environments viz.
+// baremetal and vagrant
+/*
```

Comment:
Did you check that these were being generated? I think I have top-level docs somewhere else in the source tree as well. If you could unify those, that'd be best.